app/controllers/downloads_controller.rb

```diff
@@ -13,6 +13,7 @@ def show
   end
 
   def send_content
+    return send_hls if file_desc.mime_type.first.to_s == "application/x-mpegURL" && params[:auth_token].present?
```

Comment:
We're only serving HLS if there's an auth token? Are we not using it for the viewer when the user is logged into Figgy?

Reply:
We're only re-formatting the HLS if there's an auth token. Otherwise it just falls through to nginx and sends the file that's on disk, unmodified.

Reply:
👍 OK, that makes sense — `send_hls` below is rewriting the existing HLS file with auth_token links.